README.md

```diff
@@ -6,16 +6,26 @@
 [![npm](https://img.shields.io/npm/v/devextreme-aspnet-data.svg?maxAge=43200)](https://www.npmjs.com/package/devextreme-aspnet-data)
 [![npm nojquery](https://img.shields.io/npm/v/devextreme-aspnet-data-nojquery.svg?maxAge=43200&label=npm+nojquery)](https://www.npmjs.com/package/devextreme-aspnet-data-nojquery)
 
-This extension enables [DevExtreme client-side widgets](https://js.devexpress.com) to consume data from the server in ASP.NET applications. A widget communicates with the server in the following manner: the widget sends data loading options (filtering, grouping, sorting, and other options) to the server, the server processes data according to these options and then sends processed data back to the widget. In this way, you delegate all intensive data operations from the client to the server, thus significantly improving the performance of DevExtreme widgets. This extension can be used directly with the widgets as well as with [ASP.NET MVC Controls](https://js.devexpress.com/Documentation/Guide/ASP.NET_MVC_Controls/).
+This library enables [DevExtreme client-side widgets](https://js.devexpress.com) to consume data from ASP.NET controllers and handlers. A widget communicates with the server in the following manner: 
```

Comment:
Why 'consume'?

Reply:
This word is typically used in such cases. You can search "consume data from" in Internet.

Reply:
verb: consume; 3rd person present: consumes; past tense: consumed; past participle: consumed; gerund or present participle: consuming

1) eat, drink, or ingest (food or drink).
"people consume a good deal of sugar in drinks"
synonyms: eat, eat up, devour, ingest, swallow, gobble, gobble up, wolf down, gorge oneself on, feast on

2) (of a fire) completely destroy.
"the fire spread rapidly, consuming many homes"
synonyms: destroy, demolish, lay waste, wipe out, annihilate, devastate

3) use up (a resource).
"this process consumes enormous amounts of energy"
synonyms: use, use up, utilize, expend

3) buy (goods or services).
"accounting provides measures of the economic goods and services consumed"

Reply:
I'm familiar with the context in which 'consume data' is typically used - it is used in the same way as no.3 in that list. So do you mean to say that it uses up all the data from ASP.NET controllers and handlers?

Reply:
Hello @dirk-pieterse. We use 'consume' in the same meaning as this Google search reveals: https://google.com/search?q=consume+data,+web+service. This is an idiomatic phrase. Typically used in the context of web services and APIs. Instead of 'consume', we can write 'perfrom CRUD operations using/via ASP.NET controllers and handlers'.

Reply:
idiomatic phrases are not suitable for technical texts because they have multiple interpretations (both figurative and literal) - they are ambiguous.

I would recommend: perform CRUD operations via ASP.NET controllers and handlers